src/PressContent.jsx

```diff
@@ -0,0 +1,193 @@
+    date: 'June 9, 2019',
+    source: 'Keep Blog',
+    aboveTheFold: "Ecosystem Growth & Innovation Lead Jarrell James joined a panel at CogX discussing adoption challenges, inherently bad actors with centralized companies, secure multi-party computation & real-life, tangible use cases, how data is stored and shared from a centralized corporate perspective and more.",
+    url: 'https://blog.keep.network/keep-network-june-roundup-db063f305ad9'
+},{
+    title: 'Keep developers present network component libp2p to IPFS',
+    date: 'May 6, 2019',
+    source: 'IPFS Youtube Channel',
+    aboveTheFold: "Piotr Dyraga and Raghav Gulati gave a presentation at an IPFS Weekly Call + Contributor Office Hours on one of the important components of Keep — libp2p.",
+    url: 'https://www.youtube.com/watch?v=EamY2U2Ohyk'
+},{
+    title: 'Corbin Pon interviewed by Underscore VC Co-Founder',
+    date: 'April 23, 2019',
+    source: 'Underscore VC Youtube Channel',
+    aboveTheFold: "Discussing the future of blockchain, Corbin Pon and Richard Dulude covered the possibilities that Keep enables for public blockchains, different levels of privacy and security, hopes and plans for a safer financial world with ownership certainty, and the top things hindering blockchain.",
+    url: 'https://www.youtube.com/watch?v=PfwJODGY6yQ'
+}]
+
+class PressContent extends Component {
+    state = {
+        pressEntries: allPressEntries.slice(0,10)
+    }
+
+    handleShowAll = () => {
+        this.setState({
+            pressEntries: allPressEntries
+        })
+    }
+
+    render() {
+        return (
+            <div className="press-content">
+                <PageSection id="title-container">
+                    <div className="title">
+                        <h1>Press Coverage</h1>
+                        <h3>For press inquiries, please contact us at social@keep.network</h3>
+                    </div>
+                </PageSection>
+                <PageSection id="media-kit-container">
+                    <div className="media-kit">
+                        <div className="media-kit-left">
+                            <h2>Do you want to write about us?</h2>
+                            <h3>Find everything you might need in our media kit</h3>
+                        </div>
+                        <div className="media-kit-right">
+                            <a href="media/KeepMediaKit.zip" target="_blank" rel="noopener noreferrer">
+                                <div className="media-kit-link">
+                                    <Newspaper />
+                                    <div className="media-kit-link-text">
+                                        Media Kit
+                                    </div>
+                                    <div className="media-kit-link-button">
+                                        download
+                                    </div>
+                                </div>
+                            </a>
```

Comment:
It feels weird to have a download button and not have it be the only hit area. Any reason why the button wouldn't be the only link?

In a related vein, this feels like a pretty heavy way to do the “label with icon and download button” pattern. Not super worried about that bit for the PR though.

Reply:
I"m happy to change it, but FWIW here are my justifications:

- I figured more is better wrt actionable screen area, nothing is particularly gained by making the label not actionable.
- I tend prefer verbose / easy-to-edit over concise and creative, but I'm happy to make this a bit more semantic if you want to see real button and label elements.

Reply:
I could see the actionable screen area argument, yeah. Threw VoiceOver at it and it looks like it reads properly, so no big deal.